core/dbt/contracts/graph/nodes.py

```diff
@@ -221,7 +221,7 @@ def node_info(self):
             "materialized": self.config.get("materialized"),
             "node_status": str(self._event_status.get("node_status")),
             "node_started_at": self._event_status.get("started_at"),
-            "node_finished_at": self._event_status.get("finished_at"),
+            "node_finished_at": cast_to_str(self._event_status.get("finished_at")),
```

Comment:
Why does `node_finished_at` need to be cast to string but not `node_started_at`?